scripts/genesis-test/generate-setup

```diff
@@ -90,7 +90,7 @@ parser.add_argument(
 # Genesis block approval parameters
 # https://rchain.atlassian.net/wiki/spaces/CORE/pages/485556483/Initializing+the+Blockchain+--+Protocol+for+generating+the+Genesis+block
 parser.add_argument(
-        '--required-sigs',
+        '--required-signatures',
```

Comment:
Just want to say that I don't think we are still using this file.

Reply:
Might be we should clean all stuff from `scripts` as outdated, but I suggest to postpone it for now.